app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> list:
```

Comment:
You can also add more informative annotation
```suggestion
def calculate_team_total_rating(team: list[Player]) -> list:
```